client/src/modules/dashboard/Dashboard.scss

```diff
@@ -0,0 +1,8 @@
+.dashboard {
+  &__form {
```

Comment:
Why you add form styles to dashboard .scss file? They was removed to dropdownForm. Dashboard shouldn't have form styles.